hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,7 +307,7 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter-context_menu-add & exit"'

```

Comment:
I would replace all patterns of
```nsis
Exec 'cmd.exe /C ...\WinPython Command Prompt.exe" ... & exit"'
Sleep ...
```
with
```nsis
ExecWait '"${APP_INSTDIR}\scripts\env.bat & ..."
```
Haven't tested extensively, but it should work better than the current one which leaves the cmd window open after install.